README.md

```diff
@@ -97,10 +101,12 @@ To start the simulated device follow these steps:
 
     ![simulated-device-telemetry](docs/assets/simulated-device-telemetry.png)
 
-1. Confirm you can access the local web server on `http://localhost:8080/`
+1. For http, confirm you can access the local web server on `http://localhost:8080/`
 
     ![web-server](docs/assets/web-server.png)
 
+    If you are using a different protocol, use the appropriate client tool to access the service running on the device. 
```

Comment:
Can we also include an example on how to verify when using ssh?

Reply:
Done!